Update Notes/2.1.md

```diff
@@ -13,9 +13,10 @@
 - [Added an armor overlay for the new armor slots](https://cdn.discordapp.com/attachments/832652653292027904/922399046528794634/unknown.png) 
 - Added a pet overlay that shows your active pet in your inventory
 - [Price graph for items on /ah and /bz](https://cdn.discordapp.com/attachments/896407218151366687/926968296929107999/unknown.png) - DeDiamondPro
+- [Added bingo tab in /pv](https://cdn.discordapp.com/attachments/702456294874808330/951909539383500820/unknown.png) - Lulonaut
 - Added wishing compass solver that shows target coordinates using two samples - CraftyOldMiner
 - Improved metal detector logic to solve using a single position in most cases using known locations based on Keeper coordinates - CraftyOldMiner
-### **Minor Changes:**
```

Comment:
:( i liked that colon

Reply:
thats the only colon in the entire file :slightly_smiling_face:

Reply:
soon i will spread your colon across that file as well if you dont undo that deletion :imp: